package.json

```diff
@@ -30,6 +30,30 @@
     "./src/*.js": "./src/*.js"
   },
   "main": "dist/jquery.js",
+  "scripts": {
+    "build": "npm install && npm run build-all-variants",
+    "build-all-variants": "grunt custom:slim --esm --filename=jquery.slim.module.js && grunt custom --esm --filename=jquery.module.js && grunt custom:slim --filename=jquery.slim.js && grunt custom",
+    "jenkins": "npm run test:browserless",
+    "lint:dev": "eslint .",
```

Comment:
I think you need to specify extensions here via the `--ext` option or it will only check `*.js`.

Reply:
`js, cjs, and mjs` are the default extensions. But, what do you think of moving to the new flat config? Especially since we'll only run on latest node https://eslint.org/docs/latest/use/configure/configuration-files. The old config pattern will be deprecated in version 9.

Reply:
I think all that's needed is for eslint-config-jquery to use JS instead of JSON.